### ISSUE TITLE
Video is too big for our cache, skipping.

### ISSUE THREAD

**Issue by c0xc** [Chocobozzz/PeerTube#4348]
**Describe the current behavior**

A mirror instance with three redundancy schedules, lots of free space and most videos missing is complaining that a video is apparently too big:
```
Running redundancy scheduler for strategy trending.
Video https://<ORIGIN>/videos/watch/<UUID> is too big for our cache, skipping.
```
According to the admin gui, the "trending" strategy has about 10 GB of free space and the video is about 160 MB in size. The recently-added strategy has more than 1 TB available.

At least in this case there's a log message explaining why this video isn't mirrored (for most other missing videos there's no mirror status explaining why they're missing). But the video really isn't too big.

**Steps to reproduce:**

1.
2.
3.

**Describe the expected behavior**

Video mirroring should work and for each missing video, there should be a status text somewhere.

**Additional information**

* PeerTube instance:
  * version: 3.3.0

**Comment by Chocobozzz** [comment]
Up? :)

**Comment by c0xc** [comment]
Hi, thanks for following up. There were lots of instances of this error up until a couple of days ago, so I'm not sure which one I looked at when I've opened this ticket but I guess that shouldn't matter. I'm going through a couple of them and nowhere is that log entry followed by any details. It always seems to look like this:
```
2021-07-04 03:52:02.594 ESC[32minfoESC[39m: Video https://<ORIGIN>/videos/watch/<UUID> is too big for our cache, skipping.
2021-07-04 03:52:41.498 ESC[32minfoESC[39m: Receiving inbox requests for 1 activities by https://<ORIGIN2>/accounts/peertube.
```

There are more recent cases but it hasn't happened since I've increased the disk space (from a couple hundred GB to over a TB - it seems like I have to provide much more space than I need), so I don't know what's going on. Unless you have an idea what I can check regarding those old log entries, I guess we could close the ticket and I'd reopen it if it happens again.
